libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -291,6 +297,133 @@ static uint8_t * encodeString( uint8_t * pDestination,
 static size_t getRemainingLength( TransportRecv_t recvFunc,
                                   NetworkContext_t * pNetworkContext );
 
+/**
```

Comment:
Most of this is copied from the v4_beta docs with updates for new parameters and return code names.